argus/log.py

```diff
@@ -12,14 +12,17 @@
 #    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #    License for the specific language governing permissions and limitations
 #    under the License.
+
+import os
 import logging
 
 from argus import config as argus_config
 
 CONFIG = argus_config.CONFIG
 
 
-DEFAULT_FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+DEFAULT_FORMAT = ('%(asctime)s - %(name)s - %(scenario)s - '
```

Comment:
I think it would be better having `scenario` and `os_type` as first logging arguments, something like `ScenarioGenericSmoke - Windows 10 - 01-01-2000` looks better imo.